### PR TITLE
[stable/field-exporter]: field-exporter

### PR DESCRIPTION
**Pull request by vzholudev**
## Description

  A chart to install [field-exporter](https://github.com/deliveryhero/field-exporter). This controller is used to fill the gap in [k8s-config-connector](https://github.com/GoogleCloudPlatform/k8s-config-connector) for exporting value from Config Connector managed resources into Secrets and ConfigMaps.

## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] I have read the [contribution instructions](https://github.com/deliveryhero/helm-charts#opening-a-pr), bumped chart version and regenerated the docs
- [x] Github actions are passing

### CODE REVIEW

File: stable/field-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "field-exporter.fullname" . }}-controller-manager
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: field-exporter
+    app.kubernetes.io/part-of: field-exporter
+    control-plane: controller-manager
```
Comment (max-rocket-internet) [inline stable/field-exporter/templates/deployment.yaml:9]: What is this label for?

Comment (vzholudev) [reply stable/field-exporter/templates/deployment.yaml:9]: @arjunrn or @mtahaahmed can you answer this question?

Comment (mtahaahmed) [reply stable/field-exporter/templates/deployment.yaml:9]: This helm chart was converted from kustomize. I wanted to change in the helm chart after I see it running in the cluster.

Comment (mtahaahmed) [reply stable/field-exporter/templates/deployment.yaml:9]: we can probably remove it for now

**Pull request merged**